alerts/config.go

```diff
@@ -0,0 +1,158 @@
+package alerts
+
+// Data models for care team alerts.
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/tidepool-org/platform/data/blood/glucose"
+	"github.com/tidepool-org/platform/structure"
+)
+
+// Config models a user's desired alerts.
+type Config struct {
+	OwnerID         string         `json:"ownerID" bson:"ownerID"`
+	InvitorID       string         `json:"invitorID" bson:"invitorID"`
+	UrgentLow       *WithThreshold `json:"urgentLow,omitempty" bson:"urgentLow,omitempty"`
+	Low             *Deluxe        `json:"low,omitempty" bson:"low,omitempty"`
+	High            *Deluxe        `json:"high,omitempty" bson:"high,omitempty"`
+	NotLooping      *WithDelay     `json:"notLooping,omitempty" bson:"notLooping,omitempty"`
+	NoCommunication *WithDelay     `json:"noCommunication,omitempty" bson:"noCommunication,omitempty"`
+}
+
+func (c Config) Validate(validator structure.Validator) {
+	validator.String("ownerID", &c.OwnerID).LengthGreaterThan(0)
+	validator.String("invitorID", &c.InvitorID).LengthGreaterThan(0)
```

Comment:
You should use the user id validator here https://github.com/tidepool-org/platform/blob/104a8bf918ee3e8e2298c3de29d2289c127a5a31/user/user.go#L100